src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7985,21 +7993,21 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
             {
                 MemberResolutionResult<PropertySymbol> resolutionResult = overloadResolutionResult.ValidResult;
                 PropertySymbol property = resolutionResult.Member;
-                RefKind? receiverRefKind = receiverOpt?.GetRefKind();
-                uint receiverEscapeScope = property.RequiresInstanceReceiver && receiverOpt != null
-                    ? receiverRefKind?.IsWritableReference() == true ? GetRefEscape(receiverOpt, LocalScopeDepth) : GetValEscape(receiverOpt, LocalScopeDepth)
+                RefKind? receiverRefKind = receiver.GetRefKind();
```

Comment:
This is going to conflict with my nullable interpolation PR

Reply:
Thanks for the heads up. Hopefully it hasn't flowed to the feature branch yet and can be dealt with in a separate PR.